### PR TITLE
*/*: Fix -fno-common

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild [hunk 1]
```diff
@@ -61,12 +57,12 @@ src_configure() {
...
 
-	laptop=$laptop \
-		econf \
+	econf \
 		$(use_with alsa) \
 		$(use_with doc doxygen_docs) \
 		$(use_with ibam) \
-		$(use_with oss)
+		$(use_with oss) \
+		laptop=$laptop
```
Comment (SoapGentoo) [inline app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild:65]:
```suggestion
		laptop="${laptop}"
```

File: app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild [hunk 2]
```diff
@@ -101,7 +97,7 @@ src_install() {
 	keepdir /etc/power/suspend.d
 	exeinto /etc/power/scripts.d
 	doexe "${FILESDIR}"/wireless
-	ln -s "${D}"/etc/power/scripts.d/wireless "${D}"/etc/power/resume.d/wireless
+	dosym ../scripts.d/wireless etc/power/resume.d/wireless
```
Comment (SoapGentoo) [inline app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild:100]:
```suggestion
	dosym ../scripts.d/wireless /etc/power/resume.d/wireless
```

File: games-emulation/pcsxr/pcsxr-1.9.94_p20190306.ebuild [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit cmake xdg-utils
+inherit xdg cmake flag-o-matic
```
Comment (SoapGentoo) [inline games-emulation/pcsxr/pcsxr-1.9.94_p20190306.ebuild:5]:
```suggestion
inherit flag-o-matic xdg cmake
```

File: games-engines/gargoyle/gargoyle-2019.1.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+	DESTDIR="${D}" \
+	_BINDIR="${EPREFIX}/usr/libexec/${PN}" \
+	_APPDIR="${EPREFIX}/usr/libexec/${PN}" \
+	_LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
+	EXEMODE=755 \
+	FILEMODE=755 \
+	jam install || die
+
+	# Install config file.
+	insinto etc
```
Comment (SoapGentoo) [inline games-engines/gargoyle/gargoyle-2019.1.1.ebuild:93]:
```suggestion
	insinto /etc
```

File: games-engines/gargoyle/gargoyle-2019.1.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+			s/ \(-O.*\)\? ;/ ;/
+			a LINKFLAGS = ${LDFLAGS} ;
+			a SHRLINKFLAGS = ${LDFLAGS} ;
+		}" Jamrules || die
+
+	# Don't link against libraries used indirectly through SDL_sound.
+	sed -i -e "/GARGLKLIBS/s/-lsmpeg -lvorbisfile//g" Jamrules || die
+
+	# Convert garglk.ini to UNIX format.
+	edos2unix garglk/garglk.ini
```
Comment (SoapGentoo) [inline games-engines/gargoyle/gargoyle-2019.1.1.ebuild:60]: `edos2unix.eclass`

File: net-misc/mstpd/mstpd-0.0.9.ebuild [hunk 6]
```diff
@@ -24,3 +24,9 @@ src_configure() {
 		--with-systemdunitdir="$(systemd_get_systemunitdir)" \
 		--localstatedir="${EPREFIX}"/var
 }
+
+src_install() {
+	default
+	dosym ../../sbin/bridge-stp lib/mstpctl-utils/mstpctl_restart_config
```
Comment (SoapGentoo) [inline net-misc/mstpd/mstpd-0.0.9.ebuild:30]:
```suggestion
	dosym ../../sbin/bridge-stp /lib/mstpctl-utils/mstpctl_restart_config
```

File: net-misc/mstpd/mstpd-0.0.9.ebuild [hunk 7]
```diff
@@ -24,3 +24,9 @@ src_configure() {
...
+	dosym bridge-stp sbin/mstp_restart
```
Comment (SoapGentoo) [inline net-misc/mstpd/mstpd-0.0.9.ebuild:31]:
```suggestion
	dosym bridge-stp /sbin/mstp_restart
```

File: sys-cluster/wulfware/wulfware-2.6.0.ebuild [hunk 8]
```diff
@@ -45,14 +43,19 @@ src_compile() {
...
+	emake prefix="${ED}/usr" libdir="${ED}/usr/$(get_libdir)" \
+		includedir="${ED}/usr/include" sysconfdir="${ED}/etc" \
 		install
 
 	dodoc AUTHORS ChangeLog NEWS NOTES README xmlsysd/DESIGN
 
 	# FIXME: Update to Gentoo style init script.
 	rm -r "${ED}"/etc/init.d/wulf2html || die
+
+	dosym libwulf.so.2.6.0 usr/lib64/libwulf.so.2
```
Comment (SoapGentoo) [inline sys-cluster/wulfware/wulfware-2.6.0.ebuild:55]:
```suggestion
	dosym libwulf.so.2.6.0 /usr/lib64/libwulf.so.2
```

File: www-servers/mongrel2/mongrel2-1.11.0-r1.ebuild [hunk 9]
```diff
@@ -10,20 +12,29 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
...
 
+PATCHES=(
+	"${FILESDIR}"/${P}-polarssl-platform-590512.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+)
+
+src_configure() {
+	tc-export CC
+	default
+}
```
Comment (SoapGentoo) [inline www-servers/mongrel2/mongrel2-1.11.0-r1.ebuild:30]: should be after `src_prepare`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild (hunk 1 above)
Comment (jsmolic) [reply app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild]: Done

File: app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild (hunk 2 above)
Comment (jsmolic) [reply app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild]: done

File: games-engines/gargoyle/gargoyle-2019.1.1.ebuild (hunk 4 above)
Comment (jsmolic) [reply games-engines/gargoyle/gargoyle-2019.1.1.ebuild]: done

File: games-engines/gargoyle/gargoyle-2019.1.1.ebuild (hunk 5 above)
Comment (jsmolic) [reply games-engines/gargoyle/gargoyle-2019.1.1.ebuild:60]: done

File: net-misc/mstpd/mstpd-0.0.9.ebuild (hunk 6 above)
Comment (jsmolic) [reply net-misc/mstpd/mstpd-0.0.9.ebuild]: done

File: sys-cluster/wulfware/wulfware-2.6.0.ebuild (hunk 8 above)
Comment (jsmolic) [reply sys-cluster/wulfware/wulfware-2.6.0.ebuild]: done

File: www-servers/mongrel2/mongrel2-1.11.0-r1.ebuild (hunk 9 above)
Comment (jsmolic) [reply www-servers/mongrel2/mongrel2-1.11.0-r1.ebuild:30]: Right, done

File: games-emulation/pcsxr/pcsxr-1.9.94_p20190306.ebuild (hunk 3 above)
Comment (jsmolic) [reply games-emulation/pcsxr/pcsxr-1.9.94_p20190306.ebuild]: done

File: app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild [hunk 10]
```diff
@@ -61,12 +57,12 @@ src_configure() {
 		laptop=powerbook
 	fi
```
Comment (SoapGentoo) [inline app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild:58]: declare `local laptop` before the `if`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild (hunk 10 above)
Comment (jsmolic) [reply app-laptop/pbbuttonsd/pbbuttonsd-0.8.1a.ebuild:59]: done

**Pull request closed without merging**